tests/baseline-handlebars-list/index.js

```diff
@@ -22,7 +22,9 @@
     name: 'Baseline: Handlebars List',
 
     setup: function() {
-      compiled = Handlebars.compile(template);
+      return $.getScript('http://builds.handlebarsjs.com.s3.amazonaws.com/handlebars-v2.0.0.js').then(function() {
```

Comment:
Hard coding this to doing a getScript every time prevents this from running offline. We could easily load Handlebars from a local vendored location instead.